uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/adyen/AdyenPaymentPresenter.kt

```diff
@@ -109,7 +110,8 @@ class AdyenPaymentPresenter(view: PaymentDropInMVP.Actions,
                         getPaymentMethods()
                     }
                 }
-                is Resource.Failure -> view?.showError(R.string.something_went_wrong)
+                is Resource.Failure -> view?.showError(R.string.something_went_wrong, result.error)
+                //TODO Consider using returnErrorStringOrLogoutIfRequired
```

Comment:
Do we need the TODOs here or can we raise a ticket for it if we think it needs doing?

Reply:
Having a ticket is fine but then whoever wants to do it, might need to know where to start. 
Not a big fan of TODOs if they rot in the project